src/test/java/uk/gov/companieshouse/mapper/CompanyOfficerMapperTest.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.companieshouse.mapper;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.api.model.officers.CompanyOfficerApi;
+import uk.gov.companieshouse.fixtures.DissolutionFixtures;
+import uk.gov.companieshouse.model.db.dissolution.DissolutionDirector;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class CompanyOfficerMapperTest {
+    private static final String DIRECTOR_NAME = "Jeff";
+    private static final String DIRECTOR_NAME_TWO = "Bill";
+    private static final String DIRECTOR_NAME_THREE = "Ted";
+    private static final LocalDate RESIGNATION_DATE = LocalDateTime.now().toLocalDate();
+
+    private final CompanyOfficerMapper mapper = new CompanyOfficerMapper();
+
+    @Test
+    public void mapCompanyOfficersToIsOverHalfSelected_halfDirectorsSelected_returnsTrue() {
+        final List<DissolutionDirector> selectedDirectors = DissolutionFixtures.generateDissolutionDirectorList();
+        final List<CompanyOfficerApi> companyOfficers = DissolutionFixtures.generateCompanyOfficerList();
+
+        selectedDirectors.get(0).setName(DIRECTOR_NAME);
+        companyOfficers.get(0).setName(DIRECTOR_NAME);
+        selectedDirectors.remove(1);
+
+        final boolean hasOverHalfDirectorsSelected =
+                mapper.mapCompanyOfficersToIsOverHalfSelected(companyOfficers, selectedDirectors);
+
+        assertTrue(hasOverHalfDirectorsSelected);
+    }
+
+    @Test
+    public void mapCompanyOfficersToIsOverHalfSelected_allDirectorsSelected_returnsTrue() {
+        final List<DissolutionDirector> selectedDirectors = DissolutionFixtures.generateDissolutionDirectorList();
+        final List<CompanyOfficerApi> companyOfficers = DissolutionFixtures.generateCompanyOfficerList();
+
+        selectedDirectors.get(0).setName(DIRECTOR_NAME);
+        companyOfficers.get(0).setName(DIRECTOR_NAME);
+        selectedDirectors.get(1).setName(DIRECTOR_NAME_TWO);
+        companyOfficers.get(1).setName(DIRECTOR_NAME_TWO);
+
+        final boolean hasOverHalfDirectorsSelected =
+                mapper.mapCompanyOfficersToIsOverHalfSelected(companyOfficers, selectedDirectors);
+
+        assertTrue(hasOverHalfDirectorsSelected);
+    }
+
+    @Test
+    public void mapCompanyOfficersToIsOverHalfSelected_noDirectorsSelected_returnsFalse() {
+        final List<DissolutionDirector> selectedDirectors = DissolutionFixtures.generateDissolutionDirectorList();
+        final List<CompanyOfficerApi> companyOfficers = DissolutionFixtures.generateCompanyOfficerList();
+
+        selectedDirectors.remove(1);
+        selectedDirectors.remove(0);
```

Comment:
not a fan of this. You are assuming that the fixture has 2 items. You should never rely on the fixture for your unit tests. They are just example objects that are liable to change. Instead this test should setup it's own data i.e.
```
final List<DissolutionDirector> selectedDirectors = Collections.emptyList();
```